src/frontend/src/lib/types/network.ts

```diff
@@ -1,10 +1,18 @@
+import type { OnRamperNetworkId } from '$lib/types/onramper';
+import type { AtLeastOne } from '$lib/types/utils';
+
 export type NetworkId = symbol;
 
 export type NetworkEnvironment = 'mainnet' | 'testnet';
 
-export interface Network {
+export type Network = {
```

Comment:
Leftover change or opiniated change?

Not against btw. maybe we should define an eslint rule for this but, in the oisy wallet signer library I think I enforced interface.

Reply:
ah! thank you! another thing I was always in doubt with! I will enforce `interface` then in another PR

Reply:
Done with https://github.com/dfinity/oisy-wallet/pull/2467